cmd/http-server/startcmd/start.go

```diff
@@ -132,6 +138,9 @@ type trustblocAgentJSOpts struct {
 	BlocDomain        string `json:"blocDomain,omitempty"`
 	WalletMediatorURL string `json:"walletMediatorURL,omitempty"`
 	LogLevel          string `json:"log-level,omitempty"`
+	// TODO get username from the actual registration process instead of a cmd line arg #266
+	AgentUsername string `json:"agent-username,omitempty"`
```

Comment:
This is also just a temporary measure until hub-auth is ready. I'm using the label command line argument as the "username" for the purposes of EDV storage. Later on the actual username will be used